Ah, I spoke too soon. I'll fix the following checkstyle problems before merge.

[INFO] There are 3 errors reported by Checkstyle 8.8 with /home/travis/build/DSpace/DSpace/dspace-api/../checkstyle.xml ruleset.
[ERROR] src/main/java/org/dspace/content/BundleServiceImpl.java:[148] (regexp) RegexpSingleline: Line has trailing whitespace
[ERROR] src/main/java/org/dspace/content/BundleServiceImpl.java:[151,11] (whitespace) WhitespaceAround: 'if' is not followed by whitespace.
[ERROR] src/main/java/org/dspace/content/BundleServiceImpl.java:[152,9] (blocks) LeftCurly: '{' at column 9 should be on the previous line.

--- a/dspace-api/src/main/java/org/dspace/content/BundleServiceImpl.java
+++ b/dspace-api/src/main/java/org/dspace/content/BundleServiceImpl.java
@@ -145,11 +145,10 @@ public class BundleServiceImpl extends DSpaceObjectServiceImpl<Bundle> implement
                 return;
             }
         }
-        
-        //Ensure that the last modified from the item is triggered !
+
+        // Ensure that the last modified from the item is triggered !
         Item owningItem = (Item) getParentObject(context, bundle);
-        if(owningItem != null)
-        {
+        if (owningItem != null) {
             itemService.updateLastModified(context, owningItem);
             itemService.update(context, owningItem);
         }

